### PR TITLE
Fix bug causing small duration on Firefox, updates PR #284

### PR DESCRIPTION
**Pull request by divayprakash**
### Types of changes
What types of changes does your code introduce? Check all the boxes that apply:
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Description
This one is a truly weird bug. Specifying the text size as `14px` causes the bug to happen whether I use `!important` or not. So, I just switched it to `14.01px`! 😉 

### Final checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING**](.github/CONTRIBUTING.md) guidelines.
- [x] All tests passed.

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -266,3 +266,7 @@ a {
 #audioplayer .plyr__volume--display, #audioplayer .plyr__progress--buffer {
     background-color: #e8e8e8;
 }
+
+#audioplayer .plyr__time--duration {
```
Comment (shakeelmohamed) [inline css/styles.css]: If this is truly a firefox only fix, let's only apply the style change in firefox: https://stackoverflow.com/questions/952861/targeting-only-firefox-with-css

Comment (divayprakash) [reply css/styles.css]: @shakeelmohamed Done in a108c6d

**Comment by shakeelmohamed** [comment]
@divayprakash Apply this patch, the test failures are related to linting issues.
I've updated the csslint version to fix a bug with the parser.
I tested the changes locally on FF 61 on Mac OS and it looks right to me.

```diff
diff --git a/css/styles.css b/css/styles.css
index 3554eae..d644d44 100644
--- a/css/styles.css
+++ b/css/styles.css
@@ -267,8 +267,10 @@ a {
     background-color: #e8e8e8;
 }
 
-@-moz-document url-prefix() {
+@-moz-document url-prefix("") {
+
     #audioplayer .plyr__time--duration {
         font-size: 14.01px;
     }
+
 }
```

**Comment by shakeelmohamed** [comment]
@divayprakash not quite, look at the *exact* patch I posted in my last comment. You **must** have a newline to pass CI - you should be running `npm test` locally before pushing btw. See the logs here: https://travis-ci.org/zen-audio-player/zen-audio-player.github.io/jobs/419302286

**Comment by divayprakash** [comment]
@shakeelmohamed Sorry about that, I couldn't test my changes earlier due to a botched Node install. Just fixed that and csslint passes now.

**Comment by shakeelmohamed** [comment]
Thanks @divayprakash 🎉

**Pull request merged**